pkg/engine/engine.go

```diff
@@ -115,6 +115,8 @@ func (e *Engine) Render(chrt *chart.Chart, values chartutil.Values) (map[string]
 type renderable struct {
 	// tpl is the current template.
 	tpl string
+	// path is the path of chart
+	path string
```

Comment:
Newer version of helm has a new field `basePath`, seems it can be used to serve the purpose.  WDYT?